README.md

```diff
@@ -38,6 +38,7 @@ The `node-version` input supports the following syntax:
 major versions: `12`, `14`, `16`  
 more specific versions: `10.15`, `14.2.0`, `16.3.0`  
 nvm lts syntax: `lts/erbium`, `lts/fermium`, `lts/*`  
+nvm latest version syntax: `node`  
```

Comment:
```suggestion
major versions: `12`, `14`, `16`\
more specific versions: `10.15`, `14.2.0`, `16.3.0`\
nvm lts syntax: `lts/erbium`, `lts/fermium`, `lts/*`\
nvm latest version syntax: `node`
```

Perhaps it would be useful to format this section in multiple lines to make it more readable.